src/V3Options.h

```diff
@@ -145,6 +146,7 @@ class V3Options {
     bool        m_autoflush;    // main switch: --autoflush
     bool        m_bboxSys;      // main switch: --bbox-sys
     bool        m_bboxUnsup;    // main switch: --bbox-unsup
+    bool        m_build;        // main switch: --build
```

Comment:
Think you are missing initing this and other new members in the V3Options constructor.

Reply:
I think primitive member variables are initialized in ctor. (I am changing m_noVerilate to m_verilate though).
https://github.com/verilator/verilator/blob/a1dbaf65a810220ee5b3225e3ed63bca7b64ed3f/src/V3Options.cpp#L1474
https://github.com/verilator/verilator/blob/a1dbaf65a810220ee5b3225e3ed63bca7b64ed3f/src/V3Options.cpp#L1497
https://github.com/verilator/verilator/blob/a1dbaf65a810220ee5b3225e3ed63bca7b64ed3f/src/V3Options.cpp#L1539

Reply:
POD types are not initialized in constructors, only statics.  cppcheck will give a warning if you miss these (though it is very noisy so lots of false warnings too).

Reply:
I am sorry for not clear message. I understand that PODs are not initialized in ctor.
I meant that the new member variables are initialized as the other existing member variables.
They are initialized in the ctor bdoy, not in the initialize list, but other members are also initialized in the body.
As the following link shows, m_build is initialized in V3Options::V3Options().

https://github.com/verilator/verilator/blob/a1dbaf65a810220ee5b3225e3ed63bca7b64ed3f/src/V3Options.cpp#L1467-L1474

I ran cppcheck with the following option, but it does not tell about m_build nor other uninitialized member.
> cppcheck V3Options.cpp --enable=all --std=c++03

Could you please clarify which variable is not initialized ?

Reply:
Misunderstanding, looks fine.